src/app/modules/react-migration/react-migration.controller.ts

```diff
@@ -9,8 +9,13 @@ import * as PublicFormController from '../form/public-form/public-form.controlle
 import { RedirectParams } from '../form/public-form/public-form.types'
 import * as HomeController from '../home/home.controller'
 
+export enum UiCookieValues {
+  react = 'react',
+  angular = 'angular',
```

Comment:
usually we use this casing by convention for enums:
```suggestion
  React = 'react',
  Angular = 'angular',
```

Reply:
Fixed in https://github.com/opengovsg/FormSG/pull/3894/commits/ae988d85b4ef7b3d2fef96ac00dc11ad0a2e78c6